.github/CONTRIBUTING.md

```diff
@@ -106,51 +76,50 @@ The`MONGODB_URL` will be `mongodb://USERNAME:PASSWORD@localhost:27017/DATABASE_N
 
 You can access mongo-express by visiting [localhost:8081](http://localhost:8081/).
 
-### Privileged Gateway Intents
-
-Privileged Gateway Intents must now be enabled within the Discord Developer Portal in order for your bot to function. The steps below will explain how to enable them.
-1. Log into the Discord website and navigate to the [applications page](https://discord.com/developers/applications) and select your application. Then select `Bot` under `Settings` 
-2. Scroll down to the Privileged Gateway Intents section and enable all the intents.
-
-### Inviting the Bot to Your Server
+## Environment Variables
```

Comment:
I feel this should be closer to `Setting up the Bot` as we talk about environment variables there. If users read down they may then copy env example over env, loosing bot secret etc etc. I know its obvious to some, but we can make it super clear

Reply:
Planning on keeping it where it is due to that section being very lengthy and the instructions for setting up the DB are more important first IMO. As a compromise, I have adjusted the formatting of the setting up of the bot.